### PR TITLE
review changes

### PR DESCRIPTION
**Pull request by manumewari**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -15,33 +16,27 @@
 #poster-text{
     font-size: 5vw;
     margin-bottom: 0rem;
-    animation-name: text-appear;
+    animation-name: text-appear1;
```
Comment (sharadJay) [inline css/index.css]: Is this a typo?

File: css/index.css [hunk 2]
```diff
@@ -96,25 +85,24 @@
...
 
 .card .card-body .card-text{
     text-align: center;
 }
 
 .poem-class{
-    margin-left: 2rem;
-    margin-right: 2rem;
-    padding: 2rem;
+    margin-left: 3%;
```
Comment (sharadJay) [inline css/index.css]: Why not combine left and right margins?

File: css/index.css (hunk 1 above)
Comment (manumewari) [reply css/index.css]: typo. corrected now

File: css/index.css (hunk 2 above)
Comment (manumewari) [reply css/index.css]: done

**Review by sharadJay — APPROVED** [review]

**Comment by sharadJay** [comment]
Lgtm

**Pull request merged**